iconoir.com/components/Footer.tsx

```diff
@@ -1,9 +1,28 @@
 import { PeaceHand } from 'iconoir-react';
+import { url } from 'inspector';
+import { Text } from 'react-native-svg';
```

Comment:
Extra imports at the top that we're not using.

Reply:
Removed